lib/components/map/default-map.tsx

```diff
@@ -263,12 +263,16 @@ class DefaultMap extends Component {
       ...baseLayer,
       name: getLayerName(baseLayer, config, intl)
     }))
+    const baseLayerUrls = baseLayersWithNames?.map((bl) => bl.url)
+    const baseLayerNames = baseLayersWithNames?.map((bl) => bl.name)
 
     return (
       <MapContainer>
         <BaseMap
-          // Only 1 base layer is supported at the moment
-          baseLayer={baseLayersWithNames?.[0]?.url}
+          baseLayer={
+            baseLayerUrls?.length > 1 ? baseLayerUrls : baseLayerUrls?.[0]
```

Comment:
Why can't you just pass in an array of 1 and handle it on the OTP UI side? Instead of having this accept two different types.